test/integration/vistor_can_view_items_test.rb

```diff
@@ -12,7 +12,16 @@ def create_items(num)
     create_items(3)
     visit items_path
 
+    within ".titlelatte0" do
+      assert page.has_content?('latte0')
+    end
+
+    within ".titlelatte1" do
+      assert page.has_content?('latte1')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.